cn/docs/tutorials/kubernetes-basics/deploy-intro.html

```diff
@@ -27,26 +27,26 @@
 			<div class="col-md-8">
 				<h3>Kubernetes 部署</h3>
 				<p>
-				一旦运行了 Kubernetes 集群，您可以在其上部署容器化应用程序。为此，您可以创建一个 Kubernetes <b>Deployment</b>。Deployment 负责创建和更新应用程序实例。创建 Deployment 后， Kubernetes master 会将 Deployment 创建的应用程序实例调度到集群中的各个节点。
+				一旦运行了 Kubernetes 集群，您可以在其上部署容器化应用程序。为此，您可以创建一个 Kubernetes <b>部署</b>。部署负责创建和更新应用程序实例。创建部署 后， Kubernetes master 会将部署创建的应用程序实例调度到集群中的各个节点。
```

Comment:
Same comments here, for those specific K8s names, like deployment, I suggest keep original English word, or at least keep English word together with Chinese translation. Similar situations like pod/StatefulSet/Job and such like